api/engineapi/data.py

```diff
@@ -176,6 +177,67 @@ class DropUpdatedResponse(BaseModel):
     active: bool = True
 
 
+class RegisterContractRequest(BaseModel):
+    blockchain: str
+    address: str
+    contract_type: str
+    title: Optional[str] = None
+    description: Optional[str] = None
+    image_uri: Optional[str] = None
+
+
+class RegisteredContract(BaseModel):
```

Comment:
If you will add
```python
    class Config:
        orm_mode = True
```
there would no need in function `render_registered_contract` it will automaticaly handle parsing from sqlalchemy to pydantic.

Reply:
It's a good suggestion. I will make this update after first deployment - since it's a bit urgent today.